15puzzle.py

```diff
@@ -0,0 +1,81 @@
+import random
+
+
+def create_massive():
+    num = list(range(1, 16))
+    num.append(' ')
+    mas = [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
+    for i in range(4):
+        for j in range(4):
+            mas[i][j] = random.choice(num)
+            num.remove(mas[i][j])
+    return mas
+
+
+def puzzle_field(massive):
+    print('---------------------')
+    for i in range(4):
+        for j in range(4):
+            print(f'|{massive[i][j]:3}', end=' ')
+        print('|')
+        print('---------------------')
+
+
+def coordinate(massive):
+    for y in range(4):
+        for x in range(4):
+            if massive[y][x] == ' ':
+                return x, y
+
+
+def shift(massive, x, y, x_change, y_change):
```

Comment:
You can simplify list of parameters for this function - you can get `x` and `y` from `massive`, so no need to have them as parameters